server/api/apiSpecification.yaml

```diff
@@ -0,0 +1,568 @@
+        description: The id of a robot
+        required: true
+        schema:
+          $ref: '#/components/schemas/RobotIds'
+    # GET /parser/getForId/:robotId
+    get:
+      tags:
+        - Robots
+      summary: Get the robot framework code of a specific robot
+      operationId: getRobotCode
+      responses:
+        200:
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/RobotCode'
+  /robots/{robotId}/robotName:
+    parameters:
+      - name: robotId
+        in: path
+        description: The id of a robot
+        required: true
+        schema:
+          $ref: '#/components/schemas/RobotIds'
+    # GET /renameRobot?id=78d09f66d2ed466cf20b06f7&newName=Bot+Browser
+    patch:
+      tags:
+        - Robots
+      summary: Updates the name of a robot
+      operationId: setRobotName
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/RobotNameObjects'
+        description: Object with new robot name
+        required: true
+      responses:
+        204:
+          description: No Content
+        400:
+          description: Bad Request
+  /robots/userAccess:
+    # // GET /shareRobotWithUser?userId=78d09f66d2ed466cf20b06f7&robotId=78d09f66d2ed466cf20b06f7
+    post:
+      tags:
+        - Robots
+      summary: Share a robot with a user
+      operationId: createUserAccessObject
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/UserAccessObjects'
+        description: Object that connects a user with a robot
+        required: true
+      responses:
+        201:
+          description: Created
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/UserAccessObjects'
+        400:
+          description: Bad Request
+  /robots/{userId}:
+    parameters:
+      - name: userId
+        in: path
+        description: The id of a user
+        required: true
+        schema:
+          $ref: '#/components/schemas/ObjectIds'
+    # GET ssot/getAvailableRobotsForUser/78d09f66d2ed466cf20b06f7
+    get:
+      tags:
+        - Robots
+      summary: Get all robots for a specific user
+      operationId: getRobotsForUser
+      responses:
+        200:
+          description: OK
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  type: object
+                  required:
+                    - _id
+                    - robotName
+                  properties:
+                    _id:
+                      $ref: '#/components/schemas/RobotIds'
+                    robotName:
+                      $ref: '#/components/schemas/RobotNames'
+    # // GET /createNewRobot?userId=78d09f66d2ed466cf20b06f7&robotName=NewRobot
```

Comment:
please check all comments again below here or write "used to be..."

Reply:
Thats a good suggestion that will help calrify the comments 👍 I implemented it ✅